site/src/pages/WorkspacePage/Workspace.tsx

```diff
@@ -219,6 +221,12 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
           className={styles.firstColumnSpacer}
           spacing={4}
         >
+          {workspace.outdated && (
+            <Alert severity="info">
+              <AlertTitle>An update is available for your workspace</AlertTitle>
+              {updateMessage}
```

Comment:
For the `updateMessage` I would use the `AlertDetail` component. I also would add an action "Update" on it.

Reply:
there's already an update button, and a second one here would look odd. they'd be within an inch of each other, and I'm not sure I want to move the only button into the alert.

Reply:
Makes sense!